.gitignore

```diff
@@ -8,3 +8,7 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+#IDE files
+.settings
+.project
```

Comment:
I think IDE related files should be controlled by each user.
Please use gitignore_global.